README.md

```diff
@@ -1,213 +1,91 @@
-@ecl/vanilla-utility-screen-reader
-@ecl/vanilla-utility-spacing
-@ecl/vanilla-utility-typography
-@ecl/vanilla-utility-z-index
-
-### Presets (dist folder contains css and js)
-
-@ecl/preset-dev
-@ecl/preset-ec
-@ecl/preset-eu
-@ecl/preset-reset
-
-### Resources (icons and logos)
-
-@ecl/resources-ec-favicons
-@ecl/resources-ec-icons
-@ecl/resources-eu-icons
-@ecl/resources-ec-logo
-@ecl/resources-eu-logo
-@ecl/resources-ec-social-icons
-@ecl/resources-flag-icons
-@ecl/resources-social-media-icons
-
-### Themes
-
-@ecl/theme-dev
-@ecl/theme-ec
-@ecl/theme-eu
-
-### Tools
-
-@ecl/builder
-@ecl/dom-utils
-
-### Specs packages (data used in the ECL demo)
-
-@ecl/specs-component-accordion
-@ecl/specs-component-blockquote
-@ecl/specs-component-breadcrumb
-@ecl/specs-component-button
-@ecl/specs-component-card
-@ecl/specs-component-checkbox
-@ecl/specs-component-date-block
-@ecl/specs-component-datepicker
-@ecl/specs-component-description-list
-@ecl/specs-component-expandable
-@ecl/specs-component-fact-figures
-@ecl/specs-component-file-upload
-@ecl/specs-component-file
-@ecl/specs-component-footer-core
-@ecl/specs-component-footer-harmonised
-@ecl/specs-component-footer-standardised
-@ecl/specs-component-gallery
-@ecl/specs-component-hero-banner
-@ecl/specs-component-icon
-@ecl/specs-component-inpage-navigation
-@ecl/specs-component-label
-@ecl/specs-component-language-list
-@ecl/specs-component-link
-@ecl/specs-component-media-container
-@ecl/specs-component-menu
-@ecl/specs-component-message
-@ecl/specs-component-ordered-list
-@ecl/specs-component-page-banner
-@ecl/specs-component-page-header-core
-@ecl/specs-component-page-header-harmonised
-@ecl/specs-component-page-header-standardised
-@ecl/specs-component-pagination
-@ecl/specs-component-radio
-@ecl/specs-component-search-form
-@ecl/specs-component-select
-@ecl/specs-component-site-header-core
-@ecl/specs-component-site-header-harmonised
-@ecl/specs-component-site-header-standardised
-@ecl/specs-component-skip-link
-@ecl/specs-component-social-media-follow
-@ecl/specs-component-social-media-share
-@ecl/specs-component-table
-@ecl/specs-component-tag
-@ecl/specs-component-text-area
-@ecl/specs-component-text-input
-@ecl/specs-component-timeline
-@ecl/specs-component-unordered-list
-@ecl/specs-utils-typography
+# ECL v3 - Europa Component Library
+
+[![Build Status](https://drone.fpfis.eu/api/badges/ec-europa/europa-component-library/status.svg)](https://drone.fpfis.eu/ec-europa/europa-component-library)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
+
+The Europa Component Library (ECL) is a library of components applicable to all European Commission websites (hosted under ec.europa.eu domain). The library contains all available components which you can use to build your website.
```

Comment:
I'm wondering if this first section is up to date (no mention to EU, and I'm not sure that DTT is still a thing).
Maybe we could use this generic description instead: https://ec.europa.eu/component-library/